### ISSUE TITLE
Simple %run fails with error if directory with spaces is in the path

### ISSUE THREAD

**Issue by N7DR** [vatlab/sos-notebook#287]
If I create a trivial workflow with just one cell, containing (per the web example):
```
%run
print('hello world')
```
then if fails, when executed, with the error:
```
ERROR: Unrecognized command line option Book/sis-directory/.tmp_script_2edyhbdl.sos
Workflow exited with code 1
```
if the workflow has been created in a directory such that anywhere in the full path name is a directory whose name contains a space. For example, if I run the notebook from the directory: ~/level-1/ARRL Book/sos-directory/
then executing the cell will fail with the above error.

Moving exactly the same notebook to:
~/level-1/ARRL-Book/sos-directory/    [the space has been replaced with a hyphen]
causes the cell to execute correctly.

**Comment by BoPeng** [comment]
Yes, the magic boils down to the execution of a workflow from command line which fails to consider space in the path.

**Comment by BoPeng** [comment]
Thanks for the bug report. I have fixed it and released sos-notebook 0.21.9.
